### DISCUSSION TITLE
Cost Estimation for January 2023

### DISCUSSION THREAD

**Discussion by joseph-kendall** [CMSgov/price-transparency-guide#609] [Q&A] [answered]
Back in March of 2022, CMS said that the MRF could contain percentages when the regulations had stated to calculate a dollar amount. Does this same guidance apply to the Cost Estimation tool that plans need to have for January of 2023?

**Comment by shaselton-usds** [comment, marked as answer]
Hi @joseph-kendall,

A [FAQ was released (Question 1)](https://www.cms.gov/files/document/faq-part-53.pdf) to address how to handle the percentage of billed charges negotiated arrangements for the machine readable files. All other reference based arrangements still need to be calculated to a dollar amount. This guidance was specific to the machine-readable files.

For more information on how to handle percentage of billed charges for the tool, please refer to the [final rule here](https://www.federalregister.gov/d/2020-24591/p-417).

**Reply by tonicwave** [reply]
Hi @ shasleton-usds,

I'm seeing machine readable from a given plan that provides only this much detail in Additional Information field:
"The negotiated rate reported for this code is a percentage, not a dollar rate. The dollar rate is calculated using this percentage and an externally-calculated price."

The FAQ as linked includes: 
"If the plan or issuer does not use negotiated rates for reimbursement of items and services, the
plan or issuer must report derived amounts, to the extent those amounts already are calculated in
the normal course of business. The TiC Final Rules do not require plans or issuers to develop a
new methodology for providing derived amounts. If the plan or issuer uses underlying fee
schedule rates for calculating cost sharing, the plan or issuer should include the underlying fee
schedule rates in addition to the negotiated rates or derived amounts."

I should expect to be able to find the "externally-calculated price" in some fee schedule elsewhere for the plan if I expect compliance? 
Or is it ok for the plan to merely provide a percentage and then not provide the basis for the actual price?

**Reply by riashah28** [reply]
@shaselton-usds bump on this. For the shopping tool, is the advice to only return the percentage value when the rate is a percentage of billed charges, without any additional context?
